tests/e2e/rofl/tests.go

```diff
@@ -105,6 +107,18 @@ func CreateUpdateRemoveTest(ctx context.Context, env *scenario.Env) error {
 func QueryTest(ctx context.Context, env *scenario.Env) error {
 	rf := rofl.NewV1(env.Client)
 
+	// Query for stake app create.
+	s, err := rf.StakeThresholds(ctx, client.RoundLatest)
+	if err != nil {
+		return err
+	}
+	if !reflect.DeepEqual(rofl.StakeThresholds{AppCreate: &types.BaseUnits{
+		Amount:       *quantity.NewFromUint64(1_000_000_000_000),
```

Comment:
I believe the default is zero, so if you want something else you need to modify either this or [this](https://github.com/oasisprotocol/oasis-sdk/blob/main/tests/runtimes/components-ronl/src/lib.rs).

Reply:
Set to 1000 for testing it.